index.js

```diff
@@ -33,6 +33,7 @@ module.exports = {
     Spacer: Layout.Spacer,
     Navigation: Layout.Navigation,
     Content: Layout.Content,
+    MDLUpgrade: require('./lib/MDLUpgrade'),
```

Comment:
But for your example, you could use the component we provide as well.

Not sure the export is needed. Because if the end-user needs access to it, it means something is missing in the library ;)